### PR TITLE
Enable hot reload tests on fuchsia.

### PR DESCRIPTION
**Pull request by godofredoc**
## Description

Flutter hot reload issues are common on fuchsia platform. Enabling tests on fuchsia devicelab will help identify issues before flutter rolls to fuchsia tree.

## Tests

I added the following tests:

* Hot reload on fuchsia.

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.
- [x] The analyzer (`flutter analyze --flutter-repo`) does not report any problems on my PR.
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Did any tests fail when you ran them? Please read [Handling breaking changes].

- [x] No, no existing tests failed, so this is *not* a breaking change.
- [ ] Yes, this is a breaking change. *If not, delete the remainder of this section.*
   - [ ] I wrote a design doc: https://flutter.dev/go/template *Replace this with a link to your design doc's short link*
   - [ ] I got input from the developer relations team, specifically from: *Replace with the names of who gave advice*
   - [ ] I wrote a migration guide: *Replace with a link to your migration guide*

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Test Coverage]: https://github.com/flutter/flutter/wiki/Test-coverage-for-package%3Aflutter
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Handling breaking changes]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes

### CODE REVIEW

File: dev/bots/run_fuchsia_tests.sh [hunk 1]
```diff
@@ -88,3 +88,10 @@ cd $flutter_dir/dev/benchmarks/test_apps/stocks/
 $flutter_bin pub get
 $flutter_bin drive -v -d $device_name --target=test_driver/stock_view.dart
 echo "$(date) END:DRIVER_TEST ---------------------------------------"
+echo "$(date) START:HOT_RELOAD_TEST ---------------------------------"
+# remove all out dated .packages references
+find $flutter_dir -name ".packages" | xargs rm
+cd $flutter_dir/dev/integration_tests/flutter_gallery
+$flutter_bin pub get
```
Comment (blasten) [inline dev/bots/run_fuchsia_tests.sh:95]: fyi: `flutter run` automatically calls `pub get` unless `--no-pub` is passed to it.

Comment (godofredoc) [reply dev/bots/run_fuchsia_tests.sh:95]: pub get removed

**Comment by goderbauer** [comment]
(PR triage): Are there still plans for this PR?

**Comment by godofredoc** [comment]
Abandoning this PR as it will be implemented in its own recipe.

**Pull request closed without merging**